lighthouse-core/scripts/chromium-web-tests/README.md

```diff
@@ -0,0 +1,59 @@
+# Chromium Web Tests
```

Comment:
can we move all these under `lighthouse-core/test/chromium-web-tests/`?

The download part fits in with other scripts in here (for better or worse), but the actual test files, expectations, and runner make more sense in there than in `scripts/` (so might as move the whole thing)